build.gradle

```diff
@@ -204,8 +204,8 @@ configure(subprojects - expected_result_generator - tempto_examples) {
                     developers {
                         developer {
                             id 'thc'
-                            name 'Teradata Hadoop Center'
-                            email 'EB230060 aat teradata doot com'
```

Comment:
I wonder who it was... I was GK186013. Numbers like in prison.

Reply:
I saw this `EB230060` somewhere. I think it was team's distribution list.